services/chainstorage/src/main/java/tech/pegasys/teku/services/chainstorage/StorageService.java

```diff
@@ -111,6 +117,25 @@ protected SafeFuture<?> doStart() {
                             pruningTimingsLabelledGauge,
                             pruningActiveLabelledGauge));
               }
+              LOG.info(
+                  "Data storage mode: {}, Retained epochs {}",
+                  config.getDataStorageMode(),
+                  config.getRetainedEpochs());
```

Comment:
we probably want to be mindful of logs here... maybe either put this to debug, or only log this when we're in archive mode...
also probalby want to think about this service WRT tree storage mode..

Reply:
yes, these some leftovers I missed from a test. I'm doing a cleanup on some of those atm. the one that survive the clean up in the end they will all be at debug level

Reply:
changed this to be a more informative log but kept it at INFO level since it is only printed at startup, WDYT?